src/System.Net.Http.Rtc/pkg/System.Net.Http.Rtc.pkgproj

```diff
@@ -1,13 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I assume you plan to dead in this package? If so should we just delete the entire library?